package.json

```
@@ -9,6 +9,10 @@
     "_prettier": "prettier . '!dist/**/*'",
     "build": "ts-node --experimental-specifier-resolution=node scripts/build",
     "build:cache": "BUILD_OUTDIR=dist-cache yarn build src/*ts",
+    "docker:build": "docker build -t db-migrations-local-build:latest . && docker image prune --force --filter label=stage=build",
```

I would prefer to only do the image build here:
```suggestion
    "docker:build": "docker build -t db-migrations-local-build:latest .",
```
for the following reasons:
* as a developer, on my local environment, I'd like to have the intermediate images, in case I'm iterating on an image and I want docker to reuse the cache layers to speed up the build during my iterations. 
* to avoid surprises of this script / command doing more than would be expected (imho) from its name.